spec/index.bs

```diff
@@ -8983,7 +8983,8 @@ It must only be included by interfaces which also include those mixins.
             - Let |pipelineDescriptor| be |encoder|.{{GPURenderCommandsMixin/[[pipeline]]}}.{{GPURenderPipeline/[[descriptor]]}}.
             - For each {{GPUIndex32}} |slot| `0` to
                 |pipelineDescriptor|.{{GPURenderPipelineDescriptor/vertex}}.{{GPUVertexState/buffers}}.length:
-                - |encoder|.{{GPURenderCommandsMixin/[[vertex_buffers]]}}[|slot|] must not be `null`.
+                - If |pipelineDescriptor|.{{GPURenderPipelineDescriptor/vertex}}.{{GPUVertexState/buffers}}[|slot|] is not {{undefined}}, 
+                    |encoder|.{{GPURenderCommandsMixin/[[vertex_buffers]]}}[|slot|] must not be `null`.
```

Comment:
nits:
- GPUVertexState/buffers is a sequence of nullable so the contents are either GPUVertexBufferLayout or null (not undefined).
- [[vertex_buffers]] is an `ordered map` which defines the `map/exists` dfn for whether a member exists in it or not.


```suggestion
                - If |pipelineDescriptor|.{{GPURenderPipelineDescriptor/vertex}}.{{GPUVertexState/buffers}}[|slot|] is not `null`,
                    |encoder|.{{GPURenderCommandsMixin/[[vertex_buffers]]}}[|slot|] must [=map/exist=].
```

FYI (confusingly), `{{undefined}}` links to the IDL *type* `undefined`, not the value `undefined`. I don't think there's a linkable definition for the values `undefined` and `null`, so we just use `` `undefined` `` and `` `null` ``.